### PR TITLE
Unifies Atmos Files to One Location

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Just moves atmos files around to mostly be in one spot---almost identical file structure to TG.

Also purges z-vents, because *screw* those things.

no CL as this has literally zero impact to players.

### CODE REVIEW

File: code/game/machinery/atmo_control.dm [hunk 1]
```diff
@@ -1,741 +0,0 @@
-/obj/machinery/air_sensor
```
Comment (Fox-McCloud) [inline code/game/machinery/atmo_control.dm:1]: moved to the computer/atmos_control.dm file.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -21,8 +21,8 @@ rust_g.dll @AffectedArc07
 
 
 ### Fox-McCloud
-/code/ATMOSPHERICS/ @Fox-McCloud
 /code/controllers/subsystem/air.dm @Fox-McCloud
 /code/controllers/subsystem/garbage.dm @Fox-McCloud
 /code/modules/reagents/chemistry/holder.dm @Fox-McCloud
 /code/modules/reagents/chemistry/reagents.dm @Fox-McCloud
+/code/modules/atmospherics/ @Fox-McCloud
```
Comment (AffectedArc07) [inline .github/CODEOWNERS:28]:
```suggestion
/code/modules/atmospherics/ @Fox-McCloud
/code/modules/reagents/chemistry/holder.dm @Fox-McCloud
/code/modules/reagents/chemistry/reagents.dm @Fox-McCloud
```

**Review by AffectedArc07 — COMMENTED** [review]
Yes I am this nitpicky

**Review by AffectedArc07 — APPROVED** [review]

**Comment by AffectedArc07** [comment]
Why were these even spanned

**Pull request merged**